src/functions/my_functions.R

```diff
@@ -3,5 +3,6 @@ write_a_word <- function(parametro){
 }
 
 resolve_riddle <- function(parametro_1, parametro_2){
```

Comment:
AGrega un espacio después del `)` como te lo sugiere [esta guía ](https://style.tidyverse.org/syntax.html#spacing)de estilo
```suggestion
resolve_riddle <- function(parametro_1, parametro_2) {
```